main.js

```diff
@@ -1,6 +1,56 @@
-import './style.css'
+const APIKEY = "api_key=f4fa6a2d7a3bb9c2777369817722c765";
+const BASE_URL = "https://api.themoviedb.org/3";
+const POPULAR_MOVIES_URL = `${BASE_URL}/discover/movie?sort_by=popularity.desc&${APIKEY}`;
+const IMG = "https://image.tmdb.org/t/p/w1280";
+const SEARCH_MOVIE_URL = `${BASE_URL}/search/movie?&${APIKEY}`;
+const noresult = document.querySelector(".noresult");
+const main = document.querySelector(".main");
+const movielist = document.querySelector(".movielist");
+const form = document.querySelector(".form");
+const search = document.querySelector(".search");
 
-document.querySelector('#app').innerHTML = `
-  <h1>Hello Vite!</h1>
-  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
-`
+const getMovie = async (url) => {
+    const res = await fetch(url);
+    const data = await res.json();
+
+    if (data.results.length > 0) {
+        showMovie(data.results.slice(0, 15));
+        noresult.style.display = "none";
+        movielist.style.display = "block";
+    } else {
+        noresult.style.display = "block";
+        movielist.style.display = "none";
+    }
+};
+getMovie(POPULAR_MOVIES_URL);
+
+const showMovie = (data) => {
+    data.forEach((result) => {
```

Comment:
Function definition needs to come before function call. Please move the function accordingly. Right now this works because of hoisting. But if we use arrow notation for functions this will give errors.